pipeline/scripts/ci/upstream_build.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
Thanks for adding this script.

Can we make this as stand-alone/independent script which can fetch build information with branch CLI argument.
**Reference :** https://github.com/red-hat-storage/cephci/blob/master/pipeline/scripts/ci/cos_cli.py

The idea is to use this standalone script in upstream pipeline job to fetch build information.

Reply:
@sunilkumarn417 Thanks for the feedback 
modified the file as per the reference
```(env) sangadi@magna002:~/cephci/cephci/pipeline/scripts/ci$ python3 upstream_cli.py -h
This script helps to provide needed info for upstream specific.

Usage:
  upstream_cli.py rpm branch_name
  upstream_cli.py image branch_name

Example:
    python upstream_cli.py rpm quincy

Options:
    -h --help                       Show this screen
```
Please review.